BranchMetrics/android-branch-deep-linking-attribution
# [SDK-2034] Remove unnecessary fields from v1/url requests

## Reference
SDK-2034 -- Remove unnecessary fields from v1/url requests

## Summary
Added a section of code to `ServerRequestCreateUrl()` in `ServerRequestCrealUrl.java` to remove 6 fields from `v1/url` requests which are no longer necessary.

## Motivation

The motivation for this change is to reduce the number of fields in v1/url requests down to only what's required or used. 

## Type Of Change


## Testing Instructions

Create short url's and observe the outgoing requests. They should not contain any of the 6 removed fields. 
```
RandomizedBundleToken,
RandomizedDeviceToken,
SessionID,
AnonID,
IsHardwareIDReal,
HardwareID
```









cc @BranchMetrics/saas-sdk-devs for visibility.
